### PR TITLE
Plugin Communication (was: Messages and Notifications)

### PR DESCRIPTION
**Pull request by pryrt**
Convert the Messages and Notifications page from the old NpWiki++

### CODE REVIEW

**Comment by pryrt** [comment]
added.  thanks for the feedback.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Summary
 * [Plugins](content/docs/plugins.md)
 * [Preferences](content/docs/preferences.md)
 * [Themes](content/docs/themes.md)
+* [Plugin Development: Messages and Notifications](content/docs/messages-notifications.md)
```
Comment (donho) [inline README.md]: 1. The short title "Plugin Development" is enough IMO
2. "Plugins" is better to be followed by "Plugin Development" IMO

File: content/docs/messages-notifications.md [hunk 2]
```diff
@@ -0,0 +1,1692 @@
+---
+title: Plugin Development: Messages and Notifications
```
Comment (donho) [inline content/docs/messages-notifications.md]: The short title "Plugin Development" is enough

**Comment by pryrt** [comment]
Okay, I've reordered so it's after "Plugins".  I've renamed to "Plugin Communication", which still focuses on the "plugin" aspect, is reasonably short, but limits the scope of the content to communication (which is handled through the messages and notifications described).

**Comment by donho** [comment]
"Plugin Communication" is a well found one.

**Pull request merged**